#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_kill_warncr.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add CR to warning line #331419

@DPATCH@
diff -urNad procps-3.2.7~/skill.c procps-3.2.7/skill.c
--- procps-3.2.7~/skill.c	2004-09-09 23:49:38.000000000 +1000
+++ procps-3.2.7/skill.c	2006-06-26 22:40:17.000000000 +1000
@@ -128,7 +128,7 @@
   sprintf(buf, "/proc/%d/stat", pid); /* pid (cmd) state ppid pgrp session tty */
   fd = open(buf,O_RDONLY);
   if(fd==-1){  /* process exited maybe */
-    if(pids && w_flag) printf("WARNING: process %d could not be found.",pid);
+    if(pids && w_flag) printf("WARNING: process %d could not be found.\n",pid);
     return;
   }
   fstat(fd, &statbuf);
@@ -333,7 +333,7 @@
   }else{
     fprintf(stderr,
       "Usage:   snice [new priority] [options] process selection criteria\n"
-      "Example: snice netscape crack +7\n"
+      "Example: snice +7 netscape crack \n"
       "\n"
       "The default priority is +4. (snice +4 ...)\n"
       "Priority numbers range from +20 (slowest) to -20 (fastest).\n"
